src/main/kotlin/Main.kt

```diff
@@ -2,16 +2,19 @@ import com.elbekD.bot.Bot
 import com.elbekD.bot.server
 import com.elbekD.bot.util.AllowedUpdate
 
+const val APP_URL = "https://pullpartybot.herokuapp.com"
+const val USER_NAME = "PullPartyBot"
+const val DEFAULT_PORT = "80"
+
 fun main() {
-    val app = "https://pullpartybot.herokuapp.com"
     val token = System.getenv("TELEGRAM_TOKEN") ?: throw RuntimeException("Unable to get system variable for token")
-    val userName = "PullPartyBot"
-    val bot = Bot.createWebhook(userName,token) {
-        url = "${app}/${token}"
+    val bot = Bot.createWebhook(USER_NAME,token) {
+        url = "${APP_URL}/${token}"
         allowedUpdates = listOf(AllowedUpdate.Message)
+
         server {
             host = "0.0.0.0"
-            port = System.getenv("PORT").toInt()
+            port = (System.getenv("PORT") ?: DEFAULT_PORT).toInt()
```

Comment:
Maybe `const val DEFAULT_PORT = 80` and `port = System.getenv("PORT")?.toInt() ?: DEFAULT_PORT`?

Reply:
> Maybe `const val DEFAULT_PORT = 80` and `port = System.getenv("PORT")?.toInt() ?: DEFAULT_PORT`?

Elvis operator would work the same way after `toInt()`

Reply:
What's the difference then?

Reply:
> function of convertion would crush on null

It won't due to safe access operator `?.`: `System.getenv("PORT")?.toInt()`.

Reply:
> What's the difference then?

Let's get to type-level:

`System.getenv` returns `String?`

`toInt()` gets `String`

See the issue?

Reply:
> It won't crush due to safe access operator `?.`: `System.getenv("PORT")?.toInt()`.

Safe access operator again.

`System.getenv("PORT") -> null / PORT`

Reply:
> > function of convertion would crush on null
> 
> It won't crush due to safe access operator `?.`: `System.getenv("PORT")?.toInt()`.

Oh, didn't see what you did there